tools/nl/embeddings/input/base/sheets_svs.csv

```diff
@@ -3389,7 +3389,7 @@ sdg/ER_PTD_TERR,Average proportion of Terrestrial Key Biodiversity Areas covered
 sdg/ER_RSK_LST,Red List Index
 sdg/SH_FPL_MTMM.AGE--Y15T49__SEX--F,Proportion of females of reproductive age who have their need for family planning satisfied with modern methods
 sdg/SH_STA_ANEM.AGE--Y15T49__SEX--F,Proportion of females aged 15-49 years with anaemia
-sdg/SH_STA_MORT.SEX--F,Maternal mortality ratio
+sdg/SH_STA_MORT.SEX--F,Maternal mortalities

```

Comment:
Wonder about plural for "mortality"?  (Makes sense for "deaths")

Reply:
updated to not plural